packages/angstrom/angstrom.0.11.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/angstrom"
+bug-reports: "https://github.com/inhabitedtype/angstrom/issues"
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & >= "1.0"}
+  "alcotest" {with-test & >= "0.8.1"}
+  "bigstringaf"
+  "result"
```

Comment:
Result has been integrated with ocaml since 4.03 (the minimum version of Angstrom), so this compat layer can be removed upstream in the future (just fyi, not a blocker to merging here)